external/RSL_LITE/module_dm.F

```diff
@@ -4257,7 +4257,7 @@ SUBROUTINE force_domain_em_part2 ( grid, ngrid, pgrid, config_flags    &
             DO i = MAX(ids,ips),MIN(ide-1,ipe)
 
                DO k = 1, kpe-1
-                  grid%pb(i,k,j) = ngrid%c3h(k)*(ngrid%c1h(k)*grid%mub(i,j)+ngrid%c2h(k)) + ngrid%c4h(k) + ngrid%p_top
+                  grid%pb(i,k,j) = ngrid%c3h(k) * grid%mub(i,j) + ngrid%c4h(k) + ngrid%p_top
```

Comment:
@rsarthur 
Instead of your fix being this:
```
grid%pb(i,k,j) = ngrid%c3h(k) * grid%mub(i,j) + ngrid%c4h(k) + ngrid%p_top
```
how about:
```
grid%pb(i,k,j) = grid%c3h(k) * grid%mub(i,j) + grid%c4h(k) + grid%p_top
```

Usually, the `ngrid` refers to the nest and the `grid` refers to either the parent or nest, depending on context. If we have `grid` on the LHS, perhaps all of the other parts of the struct should be from `grid`?

Reply:
@davegill 
I see your point, and have sort of been wondering myself why `ngrid` is used here. It is my impression that in this portion of the code, `ngrid` refers to the nest and `grid` refers to the "intermediate" grid, which has the horizontal resolution of the parent and the vertical levels of the nest. Is that correct? If so, then it seems like we could make the change you suggest. 

However, `ngrid` variables are used similarly throughout this subroutine, so if we change this one line, we might want to make the change throughout. Either way, I'm happy to make the change(s) and rerun my test cases.

Note also that in previous versions of the code, prior to the addition of the hybrid coordinate, `ngrid` was used in a similar way on the RHS in this subroutine.

@kalundquist, do you have any thoughts on this?

Reply:
@kalunquist @rsarthur 
Thanks for digging in and solving these issues. It is much appreciated.

Reply:
You're welcome, @davegill. Thank you for your careful reviews. Hopefully this will allow people to use vertical nesting successfully in v4.